config/locales/helpers.en.yml

```diff
@@ -223,6 +225,8 @@ en:
           provider: Upload proof of a Secure English Language Test (SELT) at B2 level, from one of the approved providers.
       teacher_interface_further_information_request_item_text_form:
         response: Enter your response
+      teacher_interface_ghana_registration_number_form:
+        registration_number: "Enter your teacher license number"
```

Comment:
I noticed that the brackets are used in this section. It might be more consistent to remove them in this section.

Reply:
Actioned: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/2379/commits/38003732d1376f7f95196f7ad406a0ebd6b3cf7d